docs/templates/snakemake/workflow/envs/[METHOD].yaml

```diff
@@ -0,0 +1,14 @@
+# Adjust METHOD_NAME and add/remove packages as needed.
```

Comment:
Dockerfile template would be good, too

Reply:
Do you have a template ready? I never created a dockerfile myself..

Reply:
included template (base python with pandas installation).